realm/src/androidTest/java/io/realm/RealmInMemoryTest.java

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
We should never copy code. (Almost) always reuse. Only make those tests that really are different. Otherwise it makes sense to reuse.